src/client/theme-default/styles/components/vp-doc.css

```diff
@@ -384,23 +384,24 @@
   transition: opacity 0.4s;
 }
 
-.vp-doc [class*='language-']:hover > span.copy {
+.vp-doc [class*='language-']:hover > button.copy,
+.vp-doc [class*='language-'] > button.copy:focus {
```

Comment:
This thing will not hide the button even if the contents are copied (as the focus still will be on that button). Also, it messes up the styles.

![image](https://user-images.githubusercontent.com/40380293/181704295-a08f9626-4faa-4f00-8283-f3296675be18.png)

Reply:
Fixed the first part. For second one needs to fix this: (set opacity 0 when button is focused)

https://github.com/vuejs/vitepress/blob/f75cafc149655694e1223142e87ffd54b68525e9/src/client/theme-default/styles/components/vp-doc.css#L432-L434